packages/babel-parser/test/expressions/experimental/top-level-await/script/input.js

```diff
@@ -0,0 +1,2 @@
+// prettier-ignore
+await +42
```

Comment:
Can you add the same test for "unambiguous" source type?

Reply:
Addressed in https://github.com/babel/babel/pull/11923/commits/cb65cb3fc4edf48ac13fc83722222ec5d6d388dd.